src/main/kotlin/com/kotlin/boardproject/domain/comment/repository/CommentRepository.kt

```diff
@@ -1,9 +1,9 @@
-package com.kotlin.boardproject.repository
+package com.kotlin.boardproject.domain.comment.repository
 
-import com.kotlin.boardproject.common.enums.PostStatus
-import com.kotlin.boardproject.model.BasePost
-import com.kotlin.boardproject.model.Comment
-import com.kotlin.boardproject.model.User
+import com.kotlin.boardproject.global.enums.PostStatus
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end